### PR TITLE
Tools: Add Korean Translator to translation issues script

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Adds @bagjunggyu to translation script such that an issue for Korean can be automatically created

CHANGELOG: SKIP

**Context: Fixes an issue?**
Fixes: #2714 

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

Ready for review. 

**What is missing until this pull request can be merged?**

Review by @hoffie I don't know the script well enough...

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [ ] I tested my code and it does what I want
-  [ ] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [ ] I've filled all the content above

### CODE REVIEW

**Review by pljones — APPROVED** [review]

**Comment by bagjunggyu** [comment]
@ann0see it is "ko_KR" needed to be fixed.

File: tools/create-translation-issues.sh [hunk 1]
```diff
@@ -45,6 +45,7 @@ declare -A TRANSLATORS_BY_LANG
 TRANSLATORS_BY_LANG[app_de_DE]="rolamos"
 TRANSLATORS_BY_LANG[app_es_ES]="ignotus666"
 TRANSLATORS_BY_LANG[app_fr_FR]="jujudusud"
+TRANSLATORS_BY_LANG[app_kr_KR]="bagjunggyu"
```
Comment (ann0see) [inline tools/create-translation-issues.sh:48]:
```suggestion
TRANSLATORS_BY_LANG[app_ko_KR]="bagjunggyu"
```

**Comment by ann0see** [comment]
@gilgongo I saw you already edited something on the master branch directly. Is this PR some then?

**Comment by gilgongo** [comment]
Yes, I added bagjunggyu (not but to web)

**Comment by hoffie** [comment]
> Review by @hoffie I don't know the script well enough...

The script should be designed to make such changes simple. The PR looks sane, not approving due to the direct commit to master though.

I don't think we should routinely commit to master directly as those commits do not trigger the usual visibility. If there are trivial changes, we can always self-merge PRs.

**Comment by gilgongo** [comment]
So I guess we need to add "Raise PR for any new translators or changes to create-translation-issues.sh" after "Declare a full freeze of the next-release and release branch" (and before "Create a translation issue for each language with ...") in the release process.

**Comment by hoffie** [comment]
We've got *Check if [sic!] the list of translators in tools/create-translation-issues.sh. Make sure the details are up-to-date. Check against https://github.com/orgs/jamulussoftware/teams/translators and confirm that is up-to-date.* already. Maybe further improve that wording (I think it has been recently updated already).

**Pull request closed without merging**